tests/summmarizer/test_text_media_matching.py

```diff
@@ -1,4 +1,59 @@
+import pytest
```

Comment:
It will be nice to do testing for TextMediaMatcher, TextMediaMatchingHelper and TextMediaMatchingPreprocessor in separate files.

Reply:
Will update this right now!

Reply:
Updated accordingly!